### PR TITLE
Create separate class for LogIn action

### LINKED ISSUE #1: Create separate class which will be responsible for login process.
This class needs to accept WebDriver as a dependency and encapsulate all required locators.

### PR DESCRIPTION
**Pull request by DIukhymenko**
Close #1 
Login action performing in another class.
All other tests are commented because of fail. Seems like driver initializing needs to be refactored and replaced.

### CODE REVIEW

File: src/test/java/com/herokuapp/cafe/loginPage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    String userName;
+    String userPassword;
+
+    public loginPage(String userName, String userPassword) {
+        this.userName = userName;
+        this.userPassword = userPassword;
+    }
+
+    private WebDriver driver;
+    public void openLoginPage() {
```
Comment (extsoft) [inline src/test/java/com/herokuapp/cafe/loginPage.java]: `openLoginPage` has to open a page, but doesn't login. If you want login, you need another method which will use given credentials.

File: src/test/java/com/herokuapp/cafe/loginPage.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        System.setProperty("webdriver.chrome.driver", "resources/chromedriver.exe");
+        driver = new ChromeDriver();
```
Comment (extsoft) [inline src/test/java/com/herokuapp/cafe/loginPage.java]: This class has to aggregate `driver`. The test class has to manage driver creation. This class will use the already prepared driver.

**Review by extsoft — CHANGES_REQUESTED** [review]
@DIukhymenko In addition to comments below, please you proper formatting for your code. Next time I'll reject PR if the code is not formatted properly.

**Comment by DIukhymenko** [comment]
@extsoft 
Please review

**Review by extsoft — APPROVED** [review]

**Pull request merged**